src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallsController.kt

```diff
@@ -22,16 +22,19 @@ class RecallsController(
 ) {
 
   @PostMapping("/recalls")
-  @ResponseBody
   fun bookRecall(@RequestBody bookRecallRequest: BookRecallRequest) =
     ResponseEntity(
       recallRepository.save(bookRecallRequest.toRecall()).toResponse(), HttpStatus.CREATED
     )
+
+  @GetMapping("/recalls")
+  fun findAll(): MutableList<Recall> = recallRepository.findAll()
```

Comment:
Personally, would return the `BookRecallResponse` object, rather than exposing our domain objects. Maybe rename it to `RecallResponse`?

Reply:
yup - makes sense 👍🏻